packages/terra-content-container/src/ContentContainer.scss

```diff
@@ -11,30 +25,14 @@
 
     > .main {
       flex: 1 1 auto;
-
-      > .normalizer {
-        -webkit-overflow-scrolling: touch;
-        -ms-overflow-style: -ms-autohiding-scrollbar; // Ensures scrollbars do not affect width in IE10
-      }
-    }
-  }
-
-  .fill {
-    height: 100%;
-    width: 100%;
-
-    > .main {
       height: 100%;
       min-height: 50px;
       overflow: hidden;
       position: relative;
       width: 100%;
 
       > .normalizer {
-        height: 100%;
-        overflow: auto;
         position: absolute;
```

Comment:
Just curious, have we tested things without the normalizer in a while? Wasn't it needed to fix an issue with Safari? I'd be curious to see if versions released in the last year and a half corrected the issue or not, though we might still run into issues with the UIWebView regardless until we're 100% on WKWebViews.